PhpAmqpLib/Message/AMQPMessage.php

```diff
@@ -39,9 +39,9 @@ class AMQPMessage extends GenericContent
 
     /**
      * @param string $body
-     * @param null $properties
+     * @param array $properties
      */
-    public function __construct($body = '', $properties = null)
+    public function __construct($body = '', $properties = array())
```

Comment:
Ah, you are correct!  Good to know :-)  Side note, I wonder if we should make the `if ($properties)` check more explicit now that you are doing this.  Maybe `if (empty($properties))` or something like that.  IDK, just a thought.